site/webpack.dev.ts

```diff
@@ -63,7 +63,7 @@ const config: Configuration = {
     port: process.env.PORT || 8080,
     proxy: {
       "/api": {
-        target: "http://localhost:3000",
+        target: "https://dev.coder.com",
```

Comment:
@kylecarbs Is this an expected change? Broke my local development dummy login creds.

Reply:
I'm going to make this a flag so this never gets in again!